go/stats/counters.go

```diff
@@ -224,3 +224,13 @@ func NewMultiCountersFunc(name string, labels []string, f CountersFunc) *MultiCo
 	}
 	return t
 }
+
+var escaper = strings.NewReplacer(".", "\\.", "\\", "\\\\")
```

Comment:
Sounds good. I've seen regexp to be more expensive. I guess it's driven by complexity.